phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The idea of phpunit.xml.dist is to define the standard but allow people to customize. It's simple, some developers who like `colors="true"`, `verbose="true"`, `stopOnFailure="true"`, etc, and some don't. With this _schema_ they can customize their own way to run PHPUnit.

How uses that phpunit.xml.dist?

- [Symfony](https://github.com/symfony/symfony/blob/master/phpunit.xml.dist);
- [Doctrine2](https://github.com/doctrine/doctrine2/blob/master/phpunit.xml.dist);
- [Monolog](https://github.com/Seldaek/monolog/blob/master/phpunit.xml.dist);
- [PHPunit](https://github.com/sebastianbergmann/phpunit/blob/master/phpunit.xml.dist) itself.

I can list all them but that would be tiring. The projects I list are not _base_ for anything, they just want that the developers who want to contribute comfortable about how they want to run the test suite.